app/config/index.js

```diff
@@ -1,15 +1,82 @@
+const Joi = require('joi')
+
 if (process.env.NODE_ENV !== 'test') {
   require('dotenv').config()
 }
 
+const schema = Joi.object({
+  env: Joi.string().valid('development', 'test', 'production').required(),
+
+  fundingFarmingApiUri: Joi.string().uri().required(),
```

Comment:
This env var is no longer used and can be removed. Remove it from here and the config on line 75